### PR TITLE
feat: Extend publisher server and apis with channel parameter.

### PR DESCRIPTION
**Pull request by yeldiRium**
Part of #943.

### CODE REVIEW

File: lib/apis/publishMessage/http/v2/postMessage.ts [hunk 1]
```diff
@@ -12,7 +12,15 @@ const postMessage = {
   path: '',
 
   request: {
-    body: {}
+    body: {
+      type: 'object',
+      properties: {
+        channel: { type: 'string' },
```
Comment (goloroden) [inline lib/apis/publishMessage/http/v2/postMessage.ts:18]:
```suggestion
        channel: { type: 'string', minLength: 1 },
```

File: lib/apis/subscribeMessages/http/v2/getMessages.ts [hunk 2]
```diff
@@ -20,10 +20,12 @@ const getMessages = {
   },
 
   getHandler ({ messageEmitter, heartbeatInterval }: {
-    messageEmitter: SpecializedEventEmitter<object>;
+    messageEmitter: EventEmitter;
     heartbeatInterval: number;
   }): WolkenkitRequestHandler {
     return async function (req: Request, res: Response): Promise<void> {
+      const { channel } = req.params;
```
Comment (goloroden) [inline lib/apis/subscribeMessages/http/v2/getMessages.ts:27]: Shouldn't this be validated?

File: lib/runtimes/microservice/processes/domainEvent/getConfiguration.ts [hunk 3]
```diff
@@ -67,6 +67,10 @@ const getConfiguration = function (): Configuration {
       default: 3000,
       schema: portSchema
     },
+    SUBSCRIBE_MESSAGES_CHANNEL: {
+      default: 'newDomainEvent',
+      schema: { type: 'string' }
```
Comment (goloroden) [inline lib/runtimes/microservice/processes/domainEvent/getConfiguration.ts:72]:
```suggestion
      schema: { type: 'string', minLength: 1 }
```

File: lib/runtimes/microservice/processes/graphql/getConfiguration.ts [hunk 4]
```diff
@@ -64,6 +64,10 @@ const getConfiguration = function (): Configuration {
       default: 3000,
       schema: portSchema
     },
+    SUBSCRIBE_MESSAGES_CHANNEL: {
+      default: 'newDomainEvent',
+      schema: { type: 'string' }
```
Comment (goloroden) [inline lib/runtimes/microservice/processes/graphql/getConfiguration.ts:69]:
```suggestion
      schema: { type: 'string', minLength: 1 }
```

File: lib/apis/subscribeMessages/http/v2/getMessages.ts (hunk 2 above)
Comment (yeldiRium) [reply lib/apis/subscribeMessages/http/v2/getMessages.ts:27]: Path parameters are validated for existence by express. The parameters are always strings and I don't think they can have a length of zero, so we should be good here.

Comment (goloroden) [reply lib/apis/subscribeMessages/http/v2/getMessages.ts:27]: You're right, I was thinking about `req.query` … sorry, mixed this up.

File: lib/runtimes/microservice/processes/domain/getConfiguration.ts [hunk 5]
```diff
@@ -57,6 +57,10 @@ const getConfiguration = function (): Configuration {
       default: 3000,
       schema: portSchema
     },
+    PUBLISHER_CHANNEL_NEW_DOMAIN_EVENT: {
+      default: 'newDomainEvent',
+      schema: { type: 'string' }
```
Comment (goloroden) [inline lib/runtimes/microservice/processes/domain/getConfiguration.ts:62]:
```suggestion
      schema: { type: 'string', minLength: 1 }
```

**Pull request merged**